react-effects/my-app/src/List.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    async function display() {
+      try {
+        const listItems = await readItems();
+        setIsLoading(false);
+        setItems(listItems);
+      } catch (error) {
+        setError(error);
```

Comment:
This code is all correct except that you need to set isLoading to false even if there's an error so that you can display the error message.

You are seeing the error message because of a bug in the exercise. My apologies.